tocco-solr-config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+import functools
+import os
+import re
+import sys
+
+IS_ENTRY = re.compile('^\s*(?P<name>[^=]*)\s*=')
+
+
+def extract_config_from_env(env):
+    return {k[5:]: v for k, v in env.items() if k.startswith('SOLR_')}
```

Comment:
prefix = 'SOLR_'
return {k[len(prefix):]: v for k, v in env.items() if k.startswith(prefix)}